docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+    web:
+        image: nginx:1.13.7
+        ports:
+            - "8080:80"
+        volumes:
+            - ./config/nginx-default.conf:/etc/nginx/conf.d/default.conf
+        depends_on:
+            - wsgi
+    wsgi:
+        build: 
+            context: .
+            dockerfile: Dockerfile.wsgi
+            args:
+                commit: "${IMAGE_TAG}"
+        image: elifesciences/profiles_wsgi:${IMAGE_TAG}
+        volumes:
+            # TODO: this can't be done as-is, it would overwrite the venv/
```

Comment:
😢 

We'll need more in `volumes` to allow editing the actual code?

Reply:
When this is separated into two `dev` and `ci` files, I plan to add `profiles/`, `test/`, etc. as separate volumes

Reply:
Another option is we could place the `venv` folder elsewhere, to be able to mount the whole folder. Same potentially applies to `var/` however.